common/src/main/java/org/apache/comet/parquet/BatchReader.java

```diff
@@ -282,6 +282,7 @@ public void init() throws URISyntaxException, IOException {
     // Initialize missing columns and use null vectors for them
     missingColumns = new boolean[columns.size()];
     List<String[]> paths = requestedSchema.getPaths();
+    ShimFileFormat.findRowIndexColumnIndexInSchema(sparkSchema);
```

Comment:
would you mind adding a comment explaining that the result is unused but that the call has side effects?
perhaps adding `let _ = ` would help make this clear

Reply:
oops, this is Java not Rust, so ignore the `let _ =` suggestion!

Reply:
I'll add a comment in the next PR.